app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,35 @@
+ .title {
+  font-size: 20px;
+  margin-bottom: 0px;
+}
+
+.piece {
+  padding-left: 10px;
```

Comment:
Properties should be ordered margin-left, padding-bottom, padding-left, padding-top